mlflow/cli.py

```diff
@@ -96,30 +100,35 @@ def run(uri, entry_point, version, param_list, experiment_id, mode, cluster_spec
     By default, Git projects run in a new working directory with the given parameters, while
     local projects run from the project's root directory.
     """
+    if experiment_id and experiment_name:
```

Comment:
Looks like this check will not behave as intended if `experiment_id = 0` and/or `experiment_name` is the empty string. This should be `if (experiment_id is not None) and (experiment_name is not None)`.

Reply:
Oh la la! Great catch.